eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Genesis.hs

```diff
@@ -85,8 +137,10 @@ pattern AlonzoGenesis
   , agMaxValSize
   , agCollateralPercentage
   , agMaxCollateralInputs
+  , agExtraConfig
   } <-
-  ( unAlonzoGenesisWrapper ->
+  AlonzoGenesisWrapper
+    { unAlonzoGenesisWrapper =
       UpgradeAlonzoPParams
         { uappCoinsPerUTxOWord = agCoinsPerUTxOWord
         , uappCostModels = agCostModels
```

Comment:
We should change the names too, not only the type of the field in `UpgradeAlonzoPParams` and `AlonzoGenesis` pattern
```suggestion
        , uappPlutusV1CostModel = agPlutusV1CostModel
```